.github/workflows/push-checks.yml

```
@@ -100,25 +105,75 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: 12.x
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
       - name: Install dependencies
         run: npm ci
       - name: Build SVGO Action
         run: npm run build
-      - name: Move SVGO Action
-        shell: bash
+      - name: Get SVGs pre-action
+        id: original-svgs
         run: |
-          mkdir ./../action
-          mv ./lib ./action.yml ./../action
-      # Just verify that the Action can be invoked successfully ...
-      - name: Run SVGO Action
+          export SVG=$(cat test/__fixtures__/not-optimized-1.svg)
```

For clarity, it may be useful to be more explicit in the file path that these SVGs are for the end-to-end tests, e.g.:

```suggestion
          export SVG=$(cat test/end-to-end/not-optimized-1.svg)
```